### PR TITLE
Feature/cronjob

### PR DESCRIPTION
**Pull request by jbarbaso**
Added status field, setters and getters to Car entity class.
Added named query to get all the inactive cars in database.
Created ActivateCarTimer to the project.

### CODE REVIEW

File: src/main/java/com/everis/cars/timer/ActivateCarTimer.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	 */
+	@EJB
+	CarService carService;
+
+	/**
+	 * {@link EntityManager} dependency injected to the service
+	 * 
+	 * @see javax.persistence.EntityManager
+	 */
+	@PersistenceContext(unitName = "em_postgres")
```
Comment (javieraviles) [inline src/main/java/com/everis/cars/timer/ActivateCarTimer.java]: this should not be here, only PersistenceService should inject persistenceContext

File: src/main/java/com/everis/cars/timer/ActivateCarTimer.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+			entityManager.merge(car);
+			logger.info("Car with ID: "+String.valueOf(car.getId())+" was persisted");
+		});
+		logger.info("Ends the activateCars method execution");
+	}
+
+	/**
+	 * @return a {@link Car} {@link List} with false status value
+	 */
+	private List<Car> getInactiveCars() {
```
Comment (javieraviles) [inline src/main/java/com/everis/cars/timer/ActivateCarTimer.java]: this method should come from CarService. On top of that, carService should delegate the execution of the namedquery to PersistenceService.

Comment (javieraviles) [reply src/main/java/com/everis/cars/timer/ActivateCarTimer.java]: something like this inside PersistenceService?

public <T> List<T> findWithNamedQuery(final String namedQueryName) {
	return entityManager.createNamedQuery(namedQueryName).getResultList();
}

File: src/main/java/com/everis/cars/timer/ActivateCarTimer.java (hunk 1 above)
Comment (jbarbaso) [reply src/main/java/com/everis/cars/timer/ActivateCarTimer.java]: Removed PersistenceService from ActivateCarTimer.

File: src/main/java/com/everis/cars/timer/ActivateCarTimer.java (hunk 2 above)
Comment (jbarbaso) [reply src/main/java/com/everis/cars/timer/ActivateCarTimer.java]: Created a method to execute named queries from PersistenceService.
Moved getInactiveCars method from ActivateCarTimer to CarService.

**Comment by jbarbaso** [comment]
Comments solved.

**Review by javieraviles — APPROVED** [review]

**Pull request merged**